vtex-apps/store-link
# Add base schema to the blocks

#### What does this PR do? \*

#### How to test it? \*
[Link to see the schema](https://klynger--storecomponents.myvtex.com/admin/cms/site-editor)

To test the `target` prop follow the instructions

1. Open the [workspace](https://klynger--storecomponents.myvtex.com/)
2. Open the quickview
3. Click in `Mais detalhes >`

It should open the product page in another tab


#### Related to / Depends on \*
